### PR TITLE
Revert "Revert "Delete fern.sh (#137)""

### PR DESCRIPTION
**Pull request by zachkirsch**
This reverts commit 90a6beaa2143871767a3c7780a6063b874a22260.

### CODE REVIEW

File: packages/compiler/plugins/plugin-runner/src/runPlugin.ts [hunk 1]
```diff
@@ -9,44 +11,39 @@ const DOCKER_PATH_TO_IR = path.join(DOCKER_FERN_DIRECTORY, "ir.json");
...
-    pluginConfig,
     pathToWriteConfigJson,
-    pluginOutputDirectory,
-    outputPathRelativeToRootOnHost,
+    relativeWorkspacePath,
 }: runPlugin.Args): Promise<void> {
-    const configJson = {
-        outputPathRelativeToRootOnHost,
+    const config: PluginConfig = {
+        relativeWorkspacePathOnHost: relativeWorkspacePath,
```
Comment (dsinghvi) [inline packages/compiler/plugins/plugin-runner/src/runPlugin.ts:28]: we need this to be `outputPathRelativeToRootOnHost`

Comment (zachkirsch) [reply packages/compiler/plugins/plugin-runner/src/runPlugin.ts:28]: Oops thanks

Comment (zachkirsch) [reply packages/compiler/plugins/plugin-runner/src/runPlugin.ts]: wait those are different things

Comment (zachkirsch) [reply packages/compiler/plugins/plugin-runner/src/runPlugin.ts]: oh i'm tracking I think

Comment (dsinghvi) [reply packages/compiler/plugins/plugin-runner/src/runPlugin.ts]: yup they are different things -- was incorrect on what i needed at first

**Pull request merged**